### ISSUE TITLE
 missing directory

### ISSUE THREAD

**Issue by vprevosto** [ANSSI-FR/x509-parser#1]
As the `build` directory is not present in the repository, trying to build the parser with `make` will fail with the following error:

```
Fatal error: can't create build/x509-parser.o: No such file or directory
```

Simply pushing an empty file into this directory should solve the issue.

**Comment by rben-dev** [comment]
Thank you for the issue!
This should be fixed in the Makefile in commit 1cc6586.

Regards,

**Comment by vprevosto** [comment]
Yes, everything is fine now, thanks for the quick fix
